app/controllers/users_controller.rb

```diff
@@ -1,9 +1,16 @@
 class UsersController < ApplicationController
+  before_action :authenticate_user!, only: [:user_sign_out]
```

Comment:
- [ ] @Team Kindly consider removing this line of code, it`s preventing the implementation of checking if the user is authenticated before accessing the root route, you can instead add a button that signs out logged-in users.